Connectors/Rhino/Speckle.Connectors.RhinoShared/Operations/Receive/RhinoHostObjectBuilder.cs

```diff
@@ -253,7 +256,7 @@ private void PreReceiveDeepClean(string baseLayerName)
     _groupBaker.PurgeGroups(baseLayerName);
   }
 
-  private Guid BakeObject(GeometryBase obj, Base originalObject, int layerIndex)
+  private Guid BakeObject(GeometryBase obj, Base originalObject, int layerIndex, string? name = null)
```

Comment:
since we already have `originalObject` in relevant functions, I wouldn't pollute function arguments. You need to do it only in BakeObjects function and thats it

Reply:
no I can't do it there, since the original object could be a line in a display value of a civil element for example